tests/apollo/test_skvbc_view_change.py

```diff
@@ -452,6 +452,52 @@ async def write_req():
         self.assertEqual(initial_primary, current_primary,
                          "Make sure we are still on the initial view.")
 
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: c == 0, rotate_keys=True)
+    @verify_linearizability()
+    async def test_recovering_of_replicas_with_initiated_view_change(self, bft_network, tracker):
+        """
+        The goal of this test is to validate recovery to fast path after stopping all replicas during view change.
+        Test is valid only when c = 0.
+        To trigger view change we start N - (F+1) replicas. This causes fast view change because 3f+2*0+1-f-1=2f
```

Comment:
"This causes fast view change" -> "This causes View Change to be initiated, but will not be completed"

Reply:
The description has been corrected. Thanks!